PCP-1889: add disableLocalAccounts method and correct order of steps (#5035)
spectrocloud/librarium: docs/docs-content/clusters/public-cloud/azure/aks.md

6.  Under **Managed Kubernetes**, select **Azure AKS** and select your Azure AKS cluster profile.

7.  Palette displays the cluster profile layers. Review the profile layers and customize parameters as desired in the

>     YAML files that display when you select a layer.

Reviewer: Do we need to say that they need to click on Values to see the YAML?

Author: Good callout.

> 7.  Palette displays the cluster profile layers. Review the profile layers and customize parameters as desired in the
>     YAML files that display when you select a layer.

Author: ```suggestion
7.  Palette displays the cluster profile layers. Review the profile layers and customize parameters as desired in the
    YAML editor that displays when you click on **Values** after selecting a layer.
```

          - <admin-group-object-id-2>
    ```

    Additionally, if you want to disable [local accounts](https://learn.microsoft.com/en-us/azure/aks/manage-local-accounts-managed-azure-ad), add the `disableLocalAccounts: true` entry to your Kubernetes cluster profile layer within the `managedControlPlane.aadProfile` section.

    ```yaml {7}
    managedControlPlane:
      aadProfile:
        managed: true
        adminGroupObjectIDs:
          - <admin-group-object-id>
          - <admin-group-object-id-2>
        disableLocalAccounts: true

>     ```

Author: This is the new entry.
